apps/ui/src/composables/useAccount.ts

```diff
@@ -21,6 +21,8 @@ export function useAccount() {
     const account = web3.value.account;
     if (!account) return;
 
+    if (web3.value.type === 'argentx' && offchainNetworks.includes(networkId)) return;
```

Comment:
This one is to avoid loading votes for starknet addresses from the hub, since it's currently returning an error, as starknet addresses are not valid voter.

This is to avoid an useless request

Reply:
Moved to another PR